src/components/atoms/Combo/Combo.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
```

Comment:
Seems like a molecule to me rather than an atom

Reply:
you did not define the prop for options correctly.
getting the following warning in console

```
 console.error node_modules/prop-types/checkPropTypes.js:20
      Warning: Failed prop type: Invalid prop `options` of type `array` supplied to `Combo`, expected `object`.
          in Combo (at Combo.stories.js:9)
          in bound Template (at Combo.test.js:8)
```